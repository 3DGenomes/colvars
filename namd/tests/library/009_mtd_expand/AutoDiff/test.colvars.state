configuration {
  step           20
  dt 5.000000e-01
}

colvar {
  name d
  x  1.42699124919382e+01
}

metadynamics {
  configuration {
  }

  hills_energy
grid_parameters {
  n_colvars 1
  lower_boundaries  0.00000000000000e+00
  upper_boundaries  1.52000000000000e+01
  widths  2.00000000000000e-01
  sizes  76
}
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 7.37198017217417e-07
 2.17725429665546e-04 5.03862848152625e-03 9.13679122397960e-03
 1.29823480653811e-03 1.44540974736741e-05 0.00000000000000e+00
 0.00000000000000e+00
  hills_energy_gradients
grid_parameters {
  n_colvars 1
  lower_boundaries  0.00000000000000e+00
  upper_boundaries  1.52000000000000e+01
  widths  2.00000000000000e-01
  sizes  76
}
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 0.00000000000000e+00
 0.00000000000000e+00 0.00000000000000e+00 2.56595520830449e-05
 4.80617343281415e-03 4.70712421636593e-02 -3.09766555642036e-02
 -2.09310717254323e-02 -4.17074600580399e-04 0.00000000000000e+00
 0.00000000000000e+00
}

