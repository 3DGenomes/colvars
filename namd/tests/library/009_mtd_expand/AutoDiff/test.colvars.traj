# step         d                     
           0    1.42467449615693e+01 
           1    1.42470958025505e+01 
           2    1.42476950281921e+01 
           3    1.42483134731004e+01 
           4    1.42492985906254e+01 
           5    1.42503801773869e+01 
           6    1.42515695462807e+01 
           7    1.42528353908450e+01 
           8    1.42542111624957e+01 
           9    1.42556786547073e+01 
          10    1.42572742488983e+01 
          11    1.42590756243960e+01 
          12    1.42606314141369e+01 
          13    1.42621978308438e+01 
          14    1.42637765729292e+01 
          15    1.42651833402819e+01 
          16    1.42663008460125e+01 
          17    1.42671222499404e+01 
          18    1.42681448349383e+01 
          19    1.42689798747923e+01 
          20    1.42699124919382e+01 
