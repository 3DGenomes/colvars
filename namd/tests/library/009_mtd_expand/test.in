colvarstrajfrequency    1
colvarsrestartfrequency 10

colvar {
    name d

    hardLowerBoundary yes
    lowerBoundary 0
    upperBoundary 10
    width 0.2
    expandBoundaries yes

    distance {
        group1 { atomnumbers 4 }
        group2 { atomnumbers 99 }
    }
}

metadynamics {
    colvars        d
}
