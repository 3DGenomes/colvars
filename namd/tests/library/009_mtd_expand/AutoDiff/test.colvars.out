colvars: ----------------------------------------------------------------------
colvars: Using NAMD interface, version 2014-10-26.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = d
colvars:   Initializing a new "distance" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:       Initializing atom group "group1".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group1" defined, 1 atoms initialized: total mass = 12.011.
colvars:       Initializing atom group "group2".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group2" defined, 1 atoms initialized: total mass = 12.011.
colvars:   All components initialized.
colvars:   # width = 0.2
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 10
colvars:   # upperWallConstant = 0 [default]
colvars:   # hardLowerBoundary = on
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = on
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "metadynamics" instance.
colvars:   # name = "metadynamics1" [default]
colvars:   # colvars = { d }
colvars:   # outputEnergy = off [default]
colvars:   # hillWeight = 0.01 [default]
colvars:   # newHillFrequency = 1000 [default]
colvars:   # hillWidth = 1.25331 [default]
colvars:   # multipleReplicas = off [default]
colvars:   # useGrids = on [default]
colvars:   # gridsUpdateFrequency = 1000 [default]
colvars:   # rebinGrids = off [default]
colvars:   Metadynamics bias "metadynamics1": Will expand grids when the colvar "d" approaches its boundaries.
colvars:   # keepHills = off [default]
colvars:   # writeFreeEnergyFile = on [default]
colvars:   # saveFreeEnergyFile = off [default]
colvars:   # writeHillsTrajectory = off [default]
colvars:   # wellTempered = off [default]
colvars:   # biasTemperature = -1 [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars:   Metadynamics bias "metadynamics1": new upper boundary for colvar "d", at 15.2.
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
